shotover-proxy/Cargo.toml

```diff
@@ -20,8 +20,8 @@ tokio-util = { version = "0.7.0", features = ["full"] }
 tokio-stream = "0.1.2"
 bytes = "1.0.0"
 bytes-utils = "0.1.1"
-futures = "0.3.12"
-futures-core = "0.3.1"
+futures = { git = "https://github.com/rust-lang/futures-rs" }
```

Comment:
This can be changed back to crates.io now

Reply:
Yep, just fixing up the merge errors.